src/resources/js/embedded-map.js

```diff
@@ -3,13 +3,14 @@
  */
 if ( "function" === typeof jQuery ) jQuery( document ).ready( function( $ ) {
 	var mapHolder,
-		position,
-		streetAddress,
-		venueCoords,
-		venueTitle;
+	    position,
+	    venueObject,
+	    venueAddress,
+	    venueCoords,
+	    venueTitle;
 
 	// The tribeEventsSingleMap object must be accessible (as it contains the venue address data etc)
-	if ("undefined" === typeof tribeEventsSingleMap ) return;
+	if ( "undefined" === typeof tribeEventsSingleMap ) return;
```

Comment:
In an age where more and more people are adopting standing desks, I feel the extra keystroke can be seen in a positive light as an integral part of a healthy and active working environment.